### DISCUSSION TITLE
How can I change the background color of the main tour card?

### DISCUSSION THREAD

**Discussion by Gxarena** [elrumordelaluz/reactour#570] [Q&A]
This is my code, I was wondering how can I change the color of the main card.

<Tour
              steps={steps}
              isOpen={isInTourMode}
              onRequestClose={() => setIsInTourMode(false)}
              accentColor="#216F00"
              highlightedMaskClassName="highlight-area" 
            />

**Comment by elrumordelaluz** [comment]
What version of `reactour` are you using?

**Reply by AtlasTimo** [reply]
Hey, I´ve the same question and I am using reactor "9.8.1". Is it the way to go to use "!important" css?

```
.onboarding {
  background-color: var(--primary-color) !important;
  color: var(--text-color) !important;
  border: var(--default-border);
  border-radius: 10px;
}

.onboarding nav {
  background-color: var(--primary-hover-color) !important;
  box-shadow: var(--default-shadow);
}

.onboarding svg {
  color: var(--placeholder-color) !important;
}

.onboarding svg:hover {
  color: var(--primary-light-hover-color) !important;
}
```

**Reply by elrumordelaluz** [reply]
try also 

```css
.onboarding.onboarding {
  /* ... */
}
```

**Reply by AtlasTimo** [reply]
Sorry. My Bad. This css code I showed is working fine. I was just wondering if this is the way to go.

**Reply by elrumordelaluz** [reply]
in `v1.x` yes.

**Reply by AtlasTimo** [reply]
I am using v2 I guess. I have installed @reactour/tour.

**Reply by elrumordelaluz** [reply]
Why not use [styles prop](https://docs.react.tours/tour/props#styles)?

[Here](https://codesandbox.io/s/currying-grass-67fhzt?file=/App.js) is an example based on [this one](https://docs.react.tours/tour/examples#custom-styles) from examples section.

**Reply by AtlasTimo** [reply]
Thanks. Didnt know there is an example section. I´ll take a look at it ^^.
